src/flamenco/types/gen_stubs.py

```diff
@@ -546,7 +546,7 @@ def __init__(self, container, json):
         self.name = json["name"]
         self.element = json["element"]
         self.compact = ("modifier" in json and json["modifier"] == "compact")
-        self.max = (json["max"] if "max" in json else None)
+        self.min = json.get("min", None)
```

Comment:
Instead of providing max instead of min, should we instead give the option to provide max OR min?

I'm a bit concerned that in the future developers may inadvertently use these data structures outside of the native programs, even if they aren't today, leading to unbounded allocations. In those cases, max should be used. Not sure how much this paranoia is justified.

Reply:
We can't stop people from mis-using data types.

Reply:
Maybe we should add this function in a follow-up PR, so we have easy-to-understand limits on the allocations?